src/views/projects/Issue.svelte

```diff
@@ -18,7 +18,7 @@
   import Markdown from "@app/components/Markdown.svelte";
   import TagInput from "./Cob/TagInput.svelte";
   import Textarea from "@app/components/Textarea.svelte";
-  import Thread from "@app/components/Thread.svelte";
+  import ThreadComponent from "@app/components/Thread.svelte";
```

Comment:
It's just for consistency, we use the same naming in `/src/views/projects/Cob/Revision.svelte`.
So I thought either we change both to `Thread` or both to `ThreadComponent`

Reply:
Ah, I think we used it like this in one of the places because we were importing a type called `Thread` and there was a clash. We should just use `Thread` in both places, since there isn't any clash.

Reply:
aae5ec7